src/Metadata/Property/Factory/SerializerPropertyMetadataFactory.php

```diff
@@ -48,8 +48,7 @@ public function create(string $resourceClass, string $property, array $options =
     {
         $propertyMetadata = $this->decorated->create($resourceClass, $property, $options);
 
-        // in case of a property inherited (in a child class), we need it's properties
-        // to be mapped against serialization groups instead of the parent ones.
+        // BC to be removed in 3.0
         if (null !== ($childResourceClass = $propertyMetadata->getChildInherited())) {
```

Comment:
can you specify that `childInherited` is deprecated which is why it needs to be removed?

Reply:
Hmm... We don't do this elsewhere. I think it's enough to know that this block should be removed when preparing 3. The code should not even be used under normal circumstances.